app/src/main/java/com/vasilevkin/catsanddogs/models/networkModels/AnimalsRemoteModel.kt

```diff
@@ -0,0 +1,8 @@
+package com.vasilevkin.catsanddogs.models.networkModels
+
+import com.google.gson.annotations.SerializedName
+
+// OLD, For dogs
+data class AnimalsRemoteModel(
+    @SerializedName("message") val photo: List<String>
```

Comment:
you should mark all types as nullable, because all data from net can be null - val photo: List<String?>? - it's a good practive to avoid NPE

Reply:
Fixed.